packages/@react-stately/collections/src/useCollection.ts

```diff
@@ -12,19 +12,16 @@
 
 import {Collection, CollectionBase, Node} from '@react-types/shared';
 import {CollectionBuilder} from './CollectionBuilder';
-import {useMemo, useRef} from 'react';
+import {useMemo} from 'react';
 
-type CollectionFactory<T, C extends Collection<Node<T>>> = (node: Iterable<Node<T>>, prev: C | null) => C;
+type CollectionFactory<T, C extends Collection<Node<T>>> = (node: Iterable<Node<T>>) => C;
```

Comment:
Technically a breaking change?

Reply:
only in so far as i removed the parameter... it wasn't being used, so don't think it's breaking. and passing something to it won't cause anything to error or break, it'll just be ignored, as it already was